e-vent.ino

```diff
@@ -230,6 +230,7 @@ void loop() {
       if (enteringState) {
         enteringState = false;
         goToPositionByDur(roboclaw, BAG_CLEAR_POS, motorPosition, tEx - (now() - tCycleTimer));
+        displ.hideIcon(3,11);
```

Comment:
Since position (row, col) information is constant, there's no need to pass it as args every time. See for example [here](https://github.com/mit-drl/e-vent/blob/419383fcbd6152720868fe15e57d1d445956a712/Display.h#L156-L164).